src/features/projects/components/ProjectSnippet.tsx

```diff
@@ -82,13 +97,41 @@ export default function ProjectSnippet({
             }}
           ></div>
         ) : null}
-
+        {project.isTopProject && project.isApproved && (
+          <div className={'topProjectBadge'}>Top Project</div>
+        )}
         <div className={'projectImageBlock'}>
           <div className={'projectType'}>
             {project.classification && t(`donate:${project.classification}`)}
           </div>
           <div className={'projectName'}>
             {truncateString(project.name, 54)}
+            {project.isApproved && (
+              <>
+                <VerifiedIcon
+                  sx={{ color: '#42A5F5' }}
+                  className={'verifiedIcon'}
+                  {...bindHover(popupState)}
+                />
+                {displayPopup && (
+                  <HoverPopover
```

Comment:
If you do now exactly move the mouse from the icon to the popup like the middle array, but rather like the other two array, the popup closes again and you are not able to click the view report link (a link which just refreshes the whole page as not containing any link, compare my other comment):

<img width="584" alt="Bildschirmfoto 2022-10-23 um 11 09 53" src="https://user-images.githubusercontent.com/1532418/197384035-bd4956ae-a40f-40a2-a20a-b640036230fc.png">

Reply:
I am not sure about the current design of displaying the month. It looked better the last time I made a screen shot showing `March 2021`:

<img width="397" alt="Bildschirm­foto 2022-11-12 um 12 50 40" src="https://user-images.githubusercontent.com/1532418/201472731-bd406430-6f90-4004-8cc0-5fa339608965.png">